Defines.php

```diff
@@ -107,17 +107,27 @@
 
 // Experimental Wikis
 if ( $cwExperimental ) {
-	$wgIncludejQueryMigrate = false;
-	if ( version_compare( MW_VERSION, '1.37', '>=' ) ) {
-		$wgParserEnableLegacyMediaDOM = false;
-	} else {
-		$wgUseNewMediaStructure = true;
-	}
+	$wgParserEnableLegacyMediaDOM = false;
+
+	$wgLocalFileRepo = [
+		'class' => LocalRepo::class,
+		'name' => 'local',
+		'directory' => $wgUploadDirectory,
+		'scriptDirUrl' => $wgScriptPath,
+		'url' => $wgUploadBaseUrl ? $wgUploadBaseUrl . $wgUploadPath : $wgUploadPath,
+		'hashLevels' => $wgHashedUploadDirectory ? 2 : 0,
+		'thumbScriptUrl' => $wgThumbnailScriptPath,
+		'transformVia404' => !$wgGenerateThumbnailOnParse,
+		'deletedDir' => $wgDeletedDirectory,
+		'deletedHashLevels' => $wgHashedUploadDirectory ? 3 : 0,
+		'updateCompatibleMetadata' => $wgUpdateCompatibleMetadata,
+		'reserializeMetadata' => $wgUpdateCompatibleMetadata,
+		'useJsonMetadata' => true,
+		'useSplitMetadata' => true,
+	];
 } else {
-	$wgIncludejQueryMigrate = true;
-	if ( version_compare( MW_VERSION, '1.37', '>=' ) ) {
-		$wgParserEnableLegacyMediaDOM = true;
-	}
+	$wgParserEnableLegacyMediaDOM = false;
```

Comment:
Should this not be added to LS, if it is set to false for regardless of experimental?